test/fuzzing/oss-fuzz-build.sh

```diff
@@ -1,3 +1,13 @@
 #/bin/bash -eu
+printf "package policy\nimport _ \"github.com/AdamKorcz/go-118-fuzz-build/testing\"\n" > $SRC/cilium/pkg/policy/registerfuzzdep.go
```

Comment:
I assume that this is done to not import fuzzing libs in policy package for building agent/whatever else might use policy package?

Reply:
This is done because OSS-Fuzz rewrites the testing types to ones compatible with libFuzzer which is the fuzzing engine used by OSS-Fuzz.